### ISSUE TITLE
Classification evaluation metric based on ontology dropout analogies

### ISSUE THREAD

**Issue by lauralorenz** [ayota/ddl_nlp#24]
**Classifying things**
Based on some information extracted from the ontology but not used to train the neural net (Don's example is, A >> B >> C, so we remove/try to classify on B / Another example is, in the ontology we may have plague is type of disease, a disease is a type of ailment, and we would remove reference to disease and leave the link between plague and ailment).

Input: csv of words with classes assigned
Output: df of word vectors and associated classes, indexed by word
